lib/menu.ts

```diff
@@ -148,41 +187,42 @@ function findAncestorChild(ancestor: Element, elem: Element|null): Element|null
 /**
  * Implements a menu item which opens a submenu.
  */
-export function menuItemSubmenu(submenu: MenuCreateFunc, ...args: DomElementArg[]): MenuItem {
+export function menuItemSubmenu(submenu: MenuCreateFunc, ...args: DomElementArg[]): Element {
   const ctl: PopupControl<IMenuOptions> = PopupControl.create(null);
 
   const popupOptions: IMenuOptions = {
     placement: 'right-start',
     trigger: ['click'],
-    attach: null,
     modifiers: {preventOverflow: {padding: 10}},
     boundaries: 'viewport',
     controller: ctl,
     isSubMenu: true,
   };
 
-  return (isSelected: Observable<boolean>) =>
-    cssMenuItem(...args, cssMenuItem.cls('-sel', isSelected),
-      dom('div', '\u25B6'),     // A right-pointing triangle
+  return cssMenuItem(...args,
+    dom('div', '\u25B6'),     // A right-pointing triangle
 
-      dom.autoDispose(ctl),
-      menu(submenu, popupOptions),
+    dom.autoDispose(ctl),
 
-      // On mouseover, open the submenu. Add a delay to avoid it on transient mouseovers.
-      dom.on('mouseover', () => ctl.open({showDelay: 250})),
+    // Set the submenu to be attached as a child of this element rather than as a sibling.
```

Comment:
Mmm, this seems fishy. It's a bit strange to have a big submenu a child of the small menuItem. It adds constraints on items (e.g. would not be OK for cssMenuItem to have css `overflow` property). But I understand the reason, since the dom children are considered selectable items. An alternative that might be cleaner is `attach: 'body'` (as the root menu), which is an option that may be set above when `popupOptions` is created. But that might interfere with some events. Would you mind to try and let me know?